packages/prometheus/README.md

```diff
@@ -0,0 +1,166 @@
+# Prometheus middleware for Hono
+
+This middleware adds basic [RED metrics](https://www.weave.works/blog/the-red-method-key-metrics-for-microservices-architecture/) to your Hono application, and exposes them on the `/metrics` endpoint for Prometheus to scrape.
+
```

Comment:
I think it would be better to have an installation guide. Users may not understand that they have to install `prom-client`.